spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,45 @@
+# : true
+
+RSpec.describe 'Post Show Page', type: :feature do
+  before do
+    # # Manually create a sample user with a post and comments
+    # @user = User.create(
+    #   username: "testuser",
+    #   profile_picture_url: "path/to/profile_picture.png",
+    # )
+    # @post = @user.posts.create(
+    #   title: "Test Post",
+    #   text: "This is the content of the test post.",
+    # )
+    # @comments = []
+    # 5.times do |i|
+    #   @comments << @post.comments.create(
+    #     content: "Comment #{i+1} on the test post.",
+    #     user: @user
+    #   )
+    # end
+
+    # visit post_path(@post)
```

Comment:
- [ ] Hi, clean code is one of the major principles we do follow at Microverse. Because Microverse needs you to become a professional software dev. I would advise you to erase all commented code.
- [ ] I would advise you to re-write your integration test, following the requirement because the test did not pass.